ci/provisioning/post_provision_config_nodes.sh

```diff
@@ -115,7 +115,7 @@ fi
 # for that device and future bindings to the device do not work, resulting in messages like, "NVMe
 # SSD [xxxx:xx:xx.x] not found" when starting daos engines.
 if lspci | grep -i nvme; then
-  daos_server storage prepare -n --reset && rmmod vfio_pci && modprobe vfio_pci
+  daos_server nvme release && rmmod vfio_pci && modprobe vfio_pci
```

Comment:
`release` or `reset`? Looks like it's supposed to be `reset`...

Reply:
can I please make the change in the follow-up PR: https://github.com/daos-stack/daos/pull/10022 so that we can make progress and get this one landed

Reply:
done in PR-10022